src/main/java/uk/sky/cqlmigrate/CassandraLockingMechanism.java

```diff
@@ -53,14 +56,17 @@ public void init() throws CannotAcquireLockException {
      * {@inheritDoc}
      * <p>
      * Returns true if successfully inserted lock.
-     * Returns false if current lock is owned by this client.
+     * Returns true if current lock is owned by this client.
      * Returns false if WriteTimeoutException thrown.
      *
      * @throws CannotAcquireLockException if any DriverException thrown while executing queries.
      */
     @Override
     public boolean acquire(String clientId) throws CannotAcquireLockException {
         try {
+            // Verify that a select of the locks completes successfully
+            // Ignore the result, we deal with key clashes during insert success/failure cases
```

Comment:
Instead of a comment, could we extract this to a method? Could call it something like `verifyClusterIsHealthy` or something similar.

Reply:
Extracted!